rel/files/ejabberd.cfg

```diff
@@ -197,6 +197,7 @@
           {"_", "/api/sse", lasse_handler, [mongoose_client_api_sse]},
           {"_", "/api/messages/[:with]", mongoose_client_api_messages, []},
           {"_", "/api/contacts/[:jid]", mongoose_client_api_contacts, []},
+          {"_", "/api/contacts", mongoose_client_api_contacts, []},
```

Comment:
Is it really necessary? `[]` in the line above should make the argument optional already.